### PR TITLE
Update due to changes in the worker

### PR DESCRIPTION
**Pull request by GeekaN2**
Changed the collector url
Added a commits field to the request

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -157,8 +157,9 @@ class HawkWebpackPlugin {
 
         body.append('file', file, {
           filename: map.name
-        })
-        body.append('release', releaseId)
+        });
+        body.append('release', releaseId);
+        body.append('commits', []);
```
Comment (neSpecc) [inline src/index.js:162]: do we need sending empty commits array?

Comment (GeekaN2) [reply src/index.js:162]: It's required field, collector won't accept release without this field

Comment (neSpecc) [reply src/index.js:162]: Will we store such an empty array in a DB?

**Review by nikmel2803 — APPROVED** [review]

**Review by neSpecc — APPROVED** [review]
Decided to make `commits` field optional later

**Pull request merged**